### ISSUE TITLE
Missing Gloomhaven Item 151 Ring of Strength

### ISSUE THREAD

**Issue by skywizeskychan** [cmlenius/gloomhaven-card-browser#40]
This is a lovely site, I love it.  

But I noticed that the item 151 is missing, Ring of Strength.

**Comment by cmlenius** [comment]
Thanks for the comment. I'm not sure which item you are reffering to though. Item 151 in gloomhaven is `Sword of the Sands`. There is an item called `Ring of Strength` which is 31 in Jaws of the Lion though.

**Comment by skywizeskychan** [comment]
woops,  That is strange.  When I looked through the entire list Item 151
was the only one I saw missing.  I'm currently playing the online version
and having a ring of strength I admit I just assumed that 151 must have
been that as I hadn't seen the ring of strength listed in there.  Now I'm
very confused as looking back at the list I do see item 151 did you add it
after my comment or am I going mad?

In either case, thank you again for your wonderful site.
